### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by mkao006**
* Implemented Docker for deployment.
* Removed price modelling from the pipeline.
* Cleaned up some minor issues.

### CODE REVIEW

File: deploy.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Deploy the Project to AWS
```
Comment (marcosmilzo) [inline deploy.md]: Thank you, @mkao006, this is great!

File: pipeline/data_harmonisation/controller.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def harmonise_article(pos_sentiment_col='positive_sentiment',
 
     sentiment_scored_article = get_sentiment_scored_article()
     topic_modelled_article = get_topic_modelled_article()
-    igc_price = get_igc_price(response='GOI')
+    igc_price = get_igc_price()
```
Comment (marcosmilzo) [inline pipeline/data_harmonisation/controller.py]: Why remove GOI?

File: requirements.txt [hunk 3]
```diff
@@ -28,81 +28,80 @@ Flask-WTF==0.12
...
 vaderSentiment==2.5
-w3lib==1.18.0
+w3lib==1.19.0
 wcwidth==0.1.7
-Werkzeug==0.12.2
+Werkzeug==0.14.1
 WTForms==2.1
 zope.deprecation==4.3.0
-zope.interface==4.4.2
+zope.interface==4.5.0
```
Comment (marcosmilzo) [inline requirements.txt]: I still have some problems when running the `setup.sh`. Somehow the versions are not corrected. I am double checking this.

File: pipeline/compute_market_force/controller.py [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+from plotly.offline import plot
+from statsmodels.nonparametric.smoothers_lowess import lowess
+
+data_dir = os.environ['DATA_DIR']
+plot_output_dir = os.environ['WEBAPP_PLOT_DIR']
+target_data_table = 'PriceForecast'
+engine = create_engine('sqlite:///{0}/the_reading_machine.db'.format(data_dir))
+harmonised_table = 'HarmonisedData'
+
+# Model parameters
```
[the remark this answers is not in the document]
Comment (mkao006) [reply pipeline/compute_market_force/controller.py]: The variables have been collected and stored under `environment.py` and `parameter.py` in the restructure branch.

https://github.com/EST-Team-Adam/TheReadingMachine/tree/restrucutre/thereadingmachine

The plan is to read them from a configuration file `thereadingmachine.cfg`, however, there are some issues with this method. Please see issue #62 .

**Pull request merged**